src/js/tech/tech.js

```diff
@@ -33,10 +34,20 @@ import document from 'global/document';
 class Tech extends Component {
 
   constructor(options={}, ready=function(){}){
+    let defaults = {
```

Comment:
we may have to set the option on videojs while we are loading HLS, but I think that we should be able to use this. Regardless the functionality was there for text tracks and this will mimic that. We may have another PR in the future to expose it further for HLS.